boards/arm/efr32mg_sltb004a/board.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2018 Christian Taedcke, Diego Sueiro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef _ASMLANGUAGE
+
+#include <soc.h>
+
+#endif /* !_ASMLANGUAGE */
+
+#ifdef __cplusplus
+}
+#endif
+
+
+/* Push button PB0 (SW0) */
+#define PB0_GPIO_NAME	CONFIG_GPIO_GECKO_PORTD_NAME
```

Comment:
Move the button and LED into device tree out of board.h

Reply:
Done